community/cloud-foundation/templates/project/README.md

```diff
@@ -1,86 +1,60 @@
     gcloud services enable deploymentmanager.googleapis.com
     gcloud services enable cloudresourcemanager.googleapis.com
     gcloud services enable cloudbilling.googleapis.com
     gcloud services enable iam.googleapis.com
     gcloud services enable servicemanagement.googleapis.com
     ```
 
-1.  **Find the *Cloud Services* service account associated with the *DM Creation Project***
+4.  Find the *Cloud Services* service account associated with the *DM Creation Project*.
 
-    It will be in the form `<project_number>@cloudservices.gserviceaccount.com`
-    and listed under [IAM & admin](https://console.cloud.google.com/iam-admin/iam)
-    in Google Cloud Console. This will be called the *"DM Service Account"* for
-    the rest of these instructions.
-    * See https://cloud.google.com/resource-manager/docs/access-control-proj
+    It is formatted as `<project_number>@cloudservices.gserviceaccount.com`,
+    and is listed under [IAM & Admin](https://console.cloud.google.com/iam-admin/iam)
+    in Google Cloud Console. This account is henceforth referred to as the *DM Service Account*. See https://cloud.google.com/resource-manager/docs/access-control-proj.
 
-1.  **Create an Organization node**
+5.  Create an Organization node.
 
-    If you don't already have an Organization node under which you will create
-    projects, then create one following [these
-    instructions](https://cloud.google.com/resource-manager/docs/creating-managing-organization).
+    If you do not already have an Organization node under which you can create
+    projects, create that node following [these instructions](https://cloud.google.com/resource-manager/docs/creating-managing-organization).
 
-1.  **Give the *DM Service Account* the following permissions on the organization node:**
+6.  Grant the *DM Service Account* the following permissions on the Organization node:
+`roles/resourcemanager.projectCreator`. This is visible in the Cloud Console's IAM permissions in *Resource Manager -> Project Creator*. See https://cloud.google.com/resource-manager/docs/access-control-proj.
 
-    * `roles/resourcemanager.projectCreator`
-        * This is visible in Cloud Console's *IAM permissions in Resource Manager
-        ->  Project Creator.*
-    * See https://cloud.google.com/resource-manager/docs/access-control-proj
-
-1.  **Create/find a *Billing Account* associated with the organization**
-
-    * See: https://cloud.google.com/support/billing/
-    * Take note if the Billing Account ID that looks like `00E12A-0AB8B2-078CE8`
-
-1.  **Give the *DM Service Account* the following permissions on the Billing Account:**
-    * `roles/billing.user`
-        * This is visible in Cloud Console's IAM permissions in
-          *Billing -> Billing Account User*.
+7.  Create/find the *Billing Account* associated with the Organization. See: https://cloud.google.com/support/billing/. Take note of the *Billing Account*'s ID, which is formatted as follows:`00E12A-0AB8B2-078CE8`.
```

Comment:
Just making sure.. this is a fake billing account right?  should we make something more obviously fake like like "012345-6789AB-CDEF01"  ?

Reply:
Note: this comment also applies to line 51 of project.py.schema

Reply:
@fawix This is a fake billing account.

Reply:
ok, I'm good then :)